apps/web/lib/withEmbedSsr.tsx

```diff
@@ -7,6 +7,28 @@ export type EmbedProps = {
 export default function withEmbedSsr(getServerSideProps: GetServerSideProps) {
   return async (context: GetServerSidePropsContext): Promise<GetServerSidePropsResult<EmbedProps>> => {
     const ssrResponse = await getServerSideProps(context);
+    const embed = context.query.embed;
+
+    if ("redirect" in ssrResponse) {
```

Comment:
Reschedule flow currently redirects /reschedule/[uid] to /pro/30min?rescheduleUid=[uid].
So, let withEmbedSsr handle redirect property to ensure that embed context is properly forwareded.